.ci/pytorch/win-test-helpers/test_custom_script_ops.py

```diff
@@ -0,0 +1,69 @@
+import os
+import subprocess
+import sys
+import contextlib
+
+
+@contextlib.contextmanager
+def pushd(new_dir):
```

Comment:
Move 7 time repeated method to helper class.

Reply:
Lies. Spend no more than 4 hours looking at the CWD param in [subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run).